src/runtime/hexagon_remote/sim_host.cpp

```diff
@@ -69,21 +69,30 @@ int init_sim() {
     }
 
     // Configue tracing.
-    const char *T = getenv("HL_HEXAGON_SIM_MIN_TRACE");
-    if (T && T[0] != 0) {
-        status = sim->SetTracing(HEX_TRACE_PC_MIN, T);
+    const char *trace = getenv("HL_HEXAGON_SIM_MIN_TRACE");
+    if (trace && trace[0] != 0) {
```

Comment:
I asked our simulator developer, and he says they are not mutually exclusive, so i will go with your approach, after all it looks nice.